lib/genWish.js

```diff
@@ -0,0 +1,17 @@
+function genWish() {
```

Comment:
make this function accept an argument (first name of the person), and accordingly modify the birthday wishes to include first name. It is more personalized.

Reply:
@thealphadollar made necessary changes.